README.md

```diff
@@ -1,4 +1,6 @@
-# Azure Functions for Rust
+# &lt;functions.rs&gt;
```

Comment:
Let's keep this "Azure Functions for Rust" for now.  That's how I refer to the project (the website I wanted shorter, so `functions.rs` made sense).